### PR TITLE
add bulbaswap volume add fees

### PR DESCRIPTION
**Pull request by madwiki**

### CODE REVIEW

**Comment by llamatester** [comment]
The bulbaswap adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts aggregators bulbaswap
    
    🦙 Running BULBASWAP adapter 🦙
    ---------------------------------------------------
    Start Date:	Mon, 04 Nov 2024 00:00:00 GMT
    End Date:	Tue, 05 Nov 2024 00:00:00 GMT
    ---------------------------------------------------
    
    MORPH 👇
    Backfill start time: 1/8/2023
    NO METHODOLOGY SPECIFIED
    End timestamp: 1730764800 (2024-11-05T00:00:00.000Z)
    Total volume: 482.72 k
    Daily volume: 3.31 k
    Daily fees: 12
    Total fees: 1.65 k

**Comment by llamatester** [comment]
The bulbaswap adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts aggregators bulbaswap
    
    🦙 Running BULBASWAP adapter 🦙
    ---------------------------------------------------
    Start Date:	Mon, 04 Nov 2024 00:00:00 GMT
    End Date:	Tue, 05 Nov 2024 00:00:00 GMT
    ---------------------------------------------------
    
    MORPH 👇
    Backfill start time: 22/10/2024
    NO METHODOLOGY SPECIFIED
    End timestamp: 1730764800 (2024-11-05T00:00:00.000Z)
    Total volume: 482.73 k
    Daily volume: 3.32 k
    Daily fees: 12
    Total fees: 1.65 k

File: aggregators/bulbaswap/index.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+}`;
+
+const v3FactoryQuery = `{
+  factory(id: "0xFf8578C2949148A6F19b7958aE86CAAb2779CDDD") {
+    totalValueLockedUSD
+    totalVolumeUSD
+    totalFeesUSD
+    txCount
+  }
+  uniswapDayDatas(first: 1, orderBy: date, orderDirection: desc) {
```
Comment (dtmkeng) [inline aggregators/bulbaswap/index.ts:33]: could you use query by Id, because if data not up to date will collect incorrect data

**Review by dtmkeng — CHANGES_REQUESTED** [review]

**Comment by dtmkeng** [comment]
thank you for PR i make some change  https://github.com/DefiLlama/dimension-adapters/pull/2095

**Pull request closed without merging**